src/features/terms-and-privacy/services.ts

```diff
@@ -0,0 +1,24 @@
+import { getStorage, removeStorage, setStorage } from '../../api/extension';
+import { STORAGE } from '../../config/config';
+
+export const getAcceptedLegalDocsVersion = async (): Promise<
+  number | undefined
+> => {
+  const version = await getStorage(STORAGE.acceptedLegalDocsVersion);
+
+  if (version) {
+    return Number(version);
+  }
+
+  return undefined;
```

Comment:
`return version ? Number (version) : undefined` ;

Reply:
Updated, thank you.
https://github.com/input-output-hk/nami/pull/911/commits/f37df1d9e8dc5717eb6bf4d9b9cf1e1f3cc12520